example/eblob_backend.c

```diff
@@ -1140,7 +1151,10 @@ static void dnet_blob_config_cleanup(struct dnet_config_backend *b)
 {
 	struct eblob_backend_config *c = b->data;
 
-	eblob_backend_cleanup(c);
+	if (c->eblob)
+		eblob_backend_cleanup(c);
+
+	free(c->data.file);
 }
```

Comment:
I've added description to `dnet_blob_config_cleanup()` and put details for monitor case.